app/views/bulk_update/recommendations_checks/show.html.erb

```diff
@@ -0,0 +1,53 @@
+<%= render PageTitle::View.new(text: "Check who you’ll recommend for QTS or EYTS") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLinkComponent.new(text: "Home", href: root_path) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full" >
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-xl"><%= current_user.organisation.name %></span>
+      Check who you’ll recommend for QTS or EYTS
+    </h1>
+
+    <% @recommendations_upload.awardable_rows.includes(:trainee).tap do |awardable_rows| %>
+      <% if awardable_rows.count <= 50 %>
```

Comment:
The number `50` is mentioned 4 times in this template. I have considered extracting - maybe settings or a constant. Holding off until I get a second opinion. There also might be a benefit of just keeping it as it is.

Reply:
I'd lean towards extracting it, just because DRY 🤷‍♂️